### ISSUE TITLE
Docs screencast is cut off on narrow browser viewports

### ISSUE THREAD

**Issue by gguillotte** [puppetlabs/wash#674]
## Describe the Change You Would Like

When viewing https://puppetlabs.github.io/wash/ on browser viewports of <1290px, I'd like to see the entire screencast view without it being cut off. Otherwise, the only way to view the screencast content is to full-screen it. This is particularly problematic because the screencast is only about 730px wide, but requires a >1280px wide viewport to view due to the site layout. This makes it difficult to play the screencast in a vertical browser window even on a large widescreen monitor.

This is especially prominent in narrower parts of the 1010-1290px viewport width range (narrow end of the desktop 64rem breakpoint) and anything narrower than 860px.

Narrower than 1290px:

![Screen Shot 2020-01-02 at 2 32 51 PM](https://user-images.githubusercontent.com/1296917/71697512-0cf09500-2d6d-11ea-895f-da159b500152.png)

Wider than 1290px:

![Screen Shot 2020-01-02 at 2 32 53 PM](https://user-images.githubusercontent.com/1296917/71697515-0eba5880-2d6d-11ea-8499-0a2554ca910e.png)

## Workarounds

Zoom out in the browser — though this reduces the text size further. I hit this issue on wider viewports because also I zoom in on the Wash docs for readability.

Use a wider viewport — which might not be an option on all devices, and makes it more difficult to have the browser and a terminal up side by side.

## Potential solutions

Have an option to view the screencast output without the asciicinema player, such as a toggleable or linked transcript. This would also make the output easier to search and copy from.

**Comment by ekinanp** [comment]
@gguillotte Thanks for filing this issue! The website's been recently updated. On mobile devices, you should be able to view all screencasts in full-screen and on narrow windows, you should be able to view the screencasts as they are. In both cases, only one part is cut-off (a `find` command), but that one part is something we explain via a caption below the screencast.

Generating transcripts for the screencasts is a bit involved and is something we'd only like to do if absolutely necessary (to the point where without them, you'd have no idea what the screencasts are showing). Please let us know if that is still the case.

**Comment by gguillotte** [comment]
@ekinanp Since the screencast width and contents were both changed since this was filed, I'll close this ticket. There are still cutoff issues, but it sounds like the new screencast's contents aren't as important as what it demonstrates.

> to the point where without them, you'd have no idea what the screencasts are about

I didn't. I instead went through the "Fundamentals" and "Filtering entries with find" tutorials, and the "Wash is a filesystem" blog post, by which point the screencast had changed. 

I understand the new screencast because I've gone through the tutorials, and the new screencast wouldn't benefit from a transcript because the explanatory context for what's happening in it is no longer embedded.

So tl;dr: I've got nothing. 🤷‍♂️
